aspnetcore/fundamentals/minimal-apis.md

```diff
@@ -1598,6 +1598,8 @@ app.MapGet("/old-path", () => Results.Redirect("/new-path"));
 app.MapGet("/download", () => Results.File("myfile.text"));
 ```
 
+<a name="binr7"></a>
```

Comment:
@Rick-Anderson  By any chance this change is to fix the _anchor_ link? `#built-in-results-1`

Reply:
Yes, guarantees it will work.

Reply:
When .NET 7 is released I'll move the .NET 7 version to the top and that will change the anchor.

Reply:
I'm almost finished with https://github.com/dotnet/AspNetCore.Docs/pull/27157/files. Really nice work.

Reply:
Don't change your PR, I submitted a suggestion to fix it.

Reply:
Thanks!!!